### ISSUE TITLE
Does custom training support setting parameters such as step and learning rate?

### ISSUE THREAD

**Issue by MrLangZe** [qubvel-org/segmentation_models.pytorch#1063]
Hi，I have another question about the mode. Can I customize the parameters such as steps and learning rate during training?

**Comment by MrLangZe** [comment]
> Hey [@MrLangZe](https://github.com/MrLangZe), you can choose any LR, please see the example

Yes, I reviewed the example code. Thanks for your reply.
